assets/models/system/Phi-4-generic-gpu/spec.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Phi-4-generic-gpu
```

Comment:
Should the name be capitalized?

Reply:
Yes - for variants where the parent model has already been published to Foundry Catalog previously, I just appended the variant suffix to the parent model name and kept the original capitalization / syntax intact.

In this case, the parent model name is (capitalized) "Phi-4", so the variant model names all start w/ "Phi-4":
![image](https://github.com/user-attachments/assets/b7fbd4e2-0950-46dc-8d03-27eaaaadf17b)